docbrown/db/benches/algobench.rs

```diff
@@ -2,40 +2,57 @@ use criterion::{criterion_group, criterion_main, Criterion};
 use docbrown_db::algorithms::local_clustering_coefficient::local_clustering_coefficient;
 use docbrown_db::algorithms::local_triangle_count::local_triangle_count;
 use docbrown_db::graph::Graph;
+use crate::common::bench;
+
+mod common;
 
 pub fn local_triangle_count_analysis(c: &mut Criterion) {
     let mut group = c.benchmark_group("local_triangle_count");
 
-    group.bench_function("local_tri_count", |b| {
-        let g: Graph = Graph::new(1);
-        let windowed_graph = g.window(0, 5);
-
-        let vs = vec![(1, 1, 2), (2, 1, 3), (3, 2, 1), (4, 3, 2)];
-
-        for (t, src, dst) in &vs {
-            g.add_edge(*t, *src, *dst, &vec![]);
+    bench(
+        &mut group,
+        "local_triangle_count",
+        None,
+        |b| {
+            let g: Graph = Graph::new(1);
+            let windowed_graph = g.window(0, 5);
+    
+            let vs = vec![(1, 1, 2), (2, 1, 3), (3, 2, 1), (4, 3, 2)];
```

Comment:
perhaps a larger number of edges? 4 seems small? 100? 200? 300? randomly (or not) generated edges?